HerPublicWebsite/appsettings.DEV.json

```diff
@@ -1,5 +1,6 @@
 {
-    "GoogleAnalytics": {
-        "MeasurementId": "G-YK22TL3X8G"
+    "BasicAuth": {
+        "Username": "hug",
+        "Password": "hug"

```

Comment:
I was thinking this wasn't _really_ that secret, although I suppose if the repo is public it's _really_ not secret...

IMO this is probably still ok, because we just want to discourage people from stumbling across the site - if they really wanted they could run it locally anyway(!) But happy to switch it back to being a proper secret.

Reply:
I think this ought to still be secret. It stops random people logging in to our test sites and confusing people doing tests with random unexpected input

Reply:
Yep, makes sense